parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -253,6 +253,14 @@ if [[ $OS == $UBUNTU_OS_NAME ]]; then
 fi
 {{end}}
 
+{{- if IsHostedMaster }}
+NCLIB="nc"
```

Comment:
You can get rid of this as we don't have to support coreos. I.e., you can just statically put nc in the command statement, it doesn't need to be in a variable